main.css

```diff
@@ -0,0 +1,800 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Lato&display=swap');
+
+* {
+  margin: 0;
+  overflow-x: hidden;
+}
+
+#back {
+  background-repeat: repeat;
+  position: absolute;
+  opacity: 0.1;
+}
+
```

Comment:
Please ensure that all page elements have similar spacing, borders, shadows, and paddings to the original design.

Reply:
resolved